src/app-root.ts

```diff
@@ -41,7 +41,7 @@ export class AppRoot extends LitElement {
 
   @state() private loggedIn: boolean = false;
 
-  @state() private actionBlob?: string;
+  @property({ type: Object, reflect: false }) latestAction?: AnalyticsEvent;
```

Comment:
q: why did we change it to a `@property` vs `@state`?

Reply:
honestly i didn't know state was reactive bc i couldn't get the value to update in DOM on change without `property`.. feel free to update back to state if u can get it to work!

Reply:
i honestly wasn't able to make it work using state... so all good